### ISSUE TITLE
correct password can't connect to ssh

### ISSUE THREAD

**Issue by drshiki** [meefik/linuxdeploy#1109]
the container is mounted successfully, but I can't connect to ssh, I tried kali,arch,debian,centos all these distros start ssh fail, only ubuntu precise version could start ssh successfully but other version also start fail.
**start ubuntu precise version ssh successfully but I can't connect to it with configured user and password, and the `terminal` in upper right corner menu doesn't open terminal but led me to http://127.0.0.1:5080, why???, I also can't find any installed file in my storage such as I search 'ssh' keyword returning no result, now I don't have any idea how to fix it, spent whole day to find the solution but not found, my device already root**, anyone any suggestion? 
so my questions are:
**1. how to fix ssh can't connect?
2. how to open terminal to use my installed ubuntu?** 

my steps is:
1. install busybox
2. install linux deploy
3. setting and configuration
4. install ubuntu
5. click configure in menu
6. start container
linux deploy version 2.3.1-247
busybox version 1.28.3-36
![C2285B8B948C9B96CBC7081171D5C6A0](https://user-images.githubusercontent.com/8835638/62916729-2d26b400-bdcc-11e9-88ec-0e4487755826.jpg)
linux deploy setting
![E46E92F4D8B7E823C8EA7939ABF9FC01](https://user-images.githubusercontent.com/8835638/62916748-4af41900-bdcc-11e9-855a-fd038a8e76f2.jpg)
![1EA2F8F72C59380106A8B8D9AFD084FD](https://user-images.githubusercontent.com/8835638/62916754-521b2700-bdcc-11e9-833d-c7da8c862326.jpg)
![4C9BCE2FB6CC84D9D75212B646E7B055](https://user-images.githubusercontent.com/8835638/62916760-57787180-bdcc-11e9-8145-369cbf47c431.jpg)
![8BCAABA19267D1C21C582629DAEAB7BD](https://user-images.githubusercontent.com/8835638/62916772-652df700-bdcc-11e9-9dab-3f7866c0e9ff.jpg)

linux deploy configuration
![FB539B37CAF2E4468BB83FC652E173C7](https://user-images.githubusercontent.com/8835638/62916787-770f9a00-bdcc-11e9-8421-bd579dfba358.jpg)
![057AB8A6D3E645476E073082D98B600E](https://user-images.githubusercontent.com/8835638/62916794-7b3bb780-bdcc-11e9-992e-68d49ec91bd4.jpg)
![FDDCBB09688C1D107420EABD25A83940](https://user-images.githubusercontent.com/8835638/62916802-81319880-bdcc-11e9-9c0c-c87f77d8aaae.jpg)
![91F56C51FA650B7EAF98243257325286](https://user-images.githubusercontent.com/8835638/62916808-87277980-bdcc-11e9-8c1f-6246dcfc42f7.jpg)
![183BC7C02950BC35E773A3DBDC8E2FBE](https://user-images.githubusercontent.com/8835638/62916812-8abb0080-bdcc-11e9-9b62-47901d552fa7.jpg)

linux deploy status
![05062C3323AE1B94520672B9E3DA302A](https://user-images.githubusercontent.com/8835638/62917028-4419d600-bdcd-11e9-819e-03b1d80cfebe.jpg)
![3DD07F8BDB5F878A87D57107EB91805A](https://user-images.githubusercontent.com/8835638/62917038-4bd97a80-bdcd-11e9-98f9-222736611eb4.jpg)

linux start debug log
![DA8AF98CDE9C99DF4965C94D89F7DE3B](https://user-images.githubusercontent.com/8835638/62917049-53008880-bdcd-11e9-9bed-c59b5cb43590.jpg)
![DE8316A871C2648FF458E55EBC0DA570](https://user-images.githubusercontent.com/8835638/62917056-598f0000-bdcd-11e9-941f-8fb0dbd8db3d.jpg)

tried busybox install directory

- /system/xbin
- /system/bin

tried busybox version

- latest in google play
- 1.28.3-36

tried user/password

- root/12345678
- root/changeme
- android/changeme
- hj/12345678
- hj/changeme

**Comment by drshiki** [comment]
> I had a problem where ssh would only work on specific distros. I can't tell if it's the same you are experiencing. I can't give you a fix but I can give you a workaround:
> 
> Don't use Linux Deploy's ssh. Install Termux and run the ssh server from there. To connect directly to your distro, use the command below.
> 
> `ssh YOUR_PHONE_IP -p8022 -t "su -c linuxdeploy shell -u USERNAME" `
Is there some ways lead me to connect my andriod phone from my computer? I want to use it on my computer

**Comment by ptrk96** [comment]
Yes, you can just run the command on my last reply on your computer. Change your phone IP and the username you chose for your installation, "hj" as I see on your screen shots.

Just one thing: I see that you're using cellular network. I don't know how that would work out. You should connect your phone and computer on the same network and use a local IP.

**Comment by snug** [comment]
Hey @ptrkc, did you ever find out what was wrong with your Linux deploy setup? Having the same problem, but thank you very much for the workaround!

**Comment by nikmax42** [comment]
Same problem

**Comment by wobbex2** [comment]
Same problem with openssh-server.
Workaround:
Uninstall openssh-server (do not uninstall openssh-sftp-server!);
Install dropbear..
Now i can connect to my phone with my filemanager:

![Bildschirmfoto](https://github.com/meefik/linuxdeploy/assets/173053814/e7e551eb-ef06-4a80-9842-59754d99c3e7)
